tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -155,7 +155,7 @@ static struct device_info boards[] = {
 		.last_sysupgrade_partition = "support-list",
 	},
 
-	/** Firmware layout for the CPE210 V2 */
+	/** Firmware layout for the CPE210 V2 and V3 */
```

Comment:
Please create an own entry for the v3. Otherwise the factory images for v3 are valid on v2 devices and vice versa.

Reply:
Yeah, I know.
I first went the route of adding a new entry but since they would be the same decided that duplicating code is not the best route.
I will edit it soon.

Yeah, I know about ath79 and have already contributed to it but I do not have access to CPE210v2 and this was tested by couple of volunteers.